packages/dashboard/src/app/views/myStories/content/index.js

```diff
@@ -88,15 +89,8 @@ function Content({
               size={THEME_CONSTANTS.TYPOGRAPHY.PRESET_SIZES.SMALL}
               as="h3"
             >
-              {search?.keyword
-                ? sprintf(
-                    /* translators: %s: search term. */
-                    __(
-                      'Sorry, we couldn\'t find any results matching "%s"',
-                      'web-stories'
-                    ),
-                    search.keyword
-                  )
+              {search?.keyword || Boolean(Object.keys(filtersObject))
+                ? __("Sorry, we couldn't find any results.", 'web-stories')
```

Comment:
```
const noResultsMessage = search?.keyword ? sprintf(
                    /* translators: %s: search term. */
                    __(
                      'Sorry, we couldn\'t find any results matching "%s"',
                      'web-stories'
                    ),
                    search.keyword
                  ) : __("Sorry, we couldn't find any results.", 'web-stories');
```

Then 
```
{search?.keyword || Boolean(Object.keys(filtersObject)) ? noResultsMessage : __('Start telling Stories.', 'web-stories')}
```

Reply:
Saw this after I started refactoring this, but this should address the logic required as well as the button logic (the button that suggests looking at templates in the case of no stories being made yet) https://github.com/GoogleForCreators/web-stories-wp/pull/11738/commits/96e74e155faf9ff15d99f24a4862b381e680192c

Reply:
If it's better to just have the idea you suggested instead of rendering the text the way I did it in the commit, let me know, it's a quick refactor to do.